modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/loader/java/JavaDeclarationDelegateTestCase.java

```diff
@@ -373,7 +373,7 @@ public void flowListeningOperationWithoutAttributes() {
 
     OperationDeclaration operation = getOperation(getConfiguration(declaration, BANANA), "getLunch");
     assertThat(operation, is(notNullValue()));
-    assertOutputType(operation.getOutput(), toMetadataType(Fruit.class), false);
+    assertOutputType(operation.getOutput(), toMetadataType(Fruit.class), true);
```

Comment:
Why did this change? Are we sure we are not breaking anything?

Reply:
The test was wrong. Because the metadata enricher is run after the declaration this wasn't noticed. Once I changed the enricher logic was moved to the loaders the test started to fail